kiosk/src/Components/MainMenu.tsx

```diff
@@ -44,7 +44,7 @@ const MainMenu: React.FC<IProps> = ({ kiosk }) => {
     return(
         <div className="mainMenu">
             <nav className={topBarClassName}>
-                <h1 className="mainMenuHeader">SELECT A GAME</h1>
+                <h1 className="mainMenuHeader">Select A Game</h1>
```

Comment:
```suggestion
                <h1 className="mainMenuHeader">Select a Game</h1>
```

nit: I feel like "a" is usually uncapitalized in title-style text (which this seems to mirror). Looks slightly better to me, but not a huge deal either way.